tests/unit/signingMessage.test.ts

```diff
@@ -0,0 +1,101 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+import { Account, Ed25519PrivateKey, generateSigningMessageForSerializable, generateSigningMessageForTransaction } from "../../src";
+import { getAptosClient } from "../e2e/helper";
+
+const { aptos } = getAptosClient();
+const TRANSFER_AMOUNT = 100;
+
+describe("generateSigningMessage ", () => {
+  const alice = Account.fromPrivateKey({
```

Comment:
lets use this file https://github.com/aptos-labs/aptos-ts-sdk/blob/main/tests/unit/helper.ts for pre-generated wallets/accounts